Dockerfile

```diff
@@ -7,5 +7,7 @@ RUN python3 -m pip install -r requirements/dev.txt
 COPY src src
 COPY test test
 
+ENV SPLUNK_HEC_TOKEN_TEST blank
```

Comment:
Would be cool if we could avoid having to set an environment variable here

Reply:
Definitely open to suggestions! This was the _easiest_ way I could add a test token from the testing host machine - without checking that token into code (which could then be used against the production Splunk to submit data to the _test_data_ index.

Reply:
If `SPLUNK_HEC_TOKEN_TEST` is set at the host (dev laptop) level to a valid token, we'll get logs in Splunk when the tests are run (via L33 in `event_handler_test.py`); however, if it's not set - the tests will still pass (L448 in `event_handler_test.py`) and not fail. Happy to share a test token and get you access to Splunk :)

Reply:
this is now commented out in the Dockerfile and set to None elsewhere which should work fine for testing, if testing with a the splunk test index, then the a token would have to be requested and set here.